keyboards/HASUPAD/config.h

```diff
@@ -0,0 +1,186 @@
+/*
+Copyright 2019 HASU
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter 
+#define VENDOR_ID       0x4C44
+#define PRODUCT_ID      0x0040
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    HASU
+#define PRODUCT         HASUPAD
+#define DESCRIPTION     A 2Encoder Keyboard
+
+*/
+#define VENDOR_ID       0x4443 // "DC" = Don Chiou
+#define PRODUCT_ID      0x5350 // "SP" = Snagpad
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    HASUObs
+#define PRODUCT         HASUPad
+#define DESCRIPTION     A 2Encoder Keyboard
```

Comment:
```suggestion
#define DESCRIPTION     A 2 Encoder Keyboard
```
might also want to change to macro pad instead of Keyboard